rpcs3/Emu/System.cpp

```diff
@@ -205,7 +205,7 @@ void Emulator::Init(bool add_only)
 		make_path_verbose(dev_hdd0 + "game/");
 		make_path_verbose(dev_hdd0 + "game/TEST12345/");
 		make_path_verbose(dev_hdd0 + "game/TEST12345/USRDIR/");
-		make_path_verbose(dev_hdd0 + "game/.locks/");
+		make_path_verbose(dev_hdd0 + reinterpret_cast<const char*>(u8"game/＄locks/"));
```

Comment:
@elad335 
The dollar sign seems wrong.
Shouldn't be $ than ＄?
then no need the u8 cast.

Reply:
Some VFS to hide files from emulation.